package.json

```diff
@@ -30,6 +38,7 @@
     "not op_mini all"
   ],
   "devDependencies": {
+    "@types/material-ui": "^0.21.6",
```

Comment:
We should prob figure out if these go in dependencies or dev dependencies

Reply:
https://devblogs.microsoft.com/typescript/the-future-of-declaration-files-2/

This blog post has them going in `dependencies`